packages/xo-server-backup-reports/src/index.js

```diff
@@ -497,18 +497,23 @@ class BackupReportsXoPlugin {
         }
 
         forEach(subTaskLog.tasks, operationLog => {
-          if (operationLog.message !== 'merge' && operationLog.message !== 'transfer') {
+          if (
+            operationLog.message !== 'merge' &&
+            operationLog.message !== 'transfer' &&
+            operationLog.message !== 'health check'
+          ) {
             return
           }
 
-          const size = operationLog.result?.size
+          const size = operationLog.result?.size ?? 0
```

Comment:
Why this change?

Reply:
I don't really like to have `NaN`   ( number + undefined ) and having to check if  `NaN > 0` or not

Reply:
This is not a NaN, it's a `undefined` here :slightly_smiling_face: (not cast to number).

We should avoid unrelated changes to avoid introducing new issues.

Also, IMHO, explicitly checking against `undefined` before comparing to `0` might be a safer course (eg in case this value was used somewhere else or will be used in the future).

Reply:
yes, I deleted the comment after re-reading the code below.